handlers.py

```diff
@@ -89,7 +89,20 @@ def birthdays(book, upcoming_days = 7):
     for item in upcoming_birthdays:
         print(f"{item['name']}: {item['congratulation_date']}")
 
+@input_error_decorator_factory(args_length = 5, message = "Invalid command. Usage: add_address <name> <street> <city> <postal_code> <country>")
```

Comment:
```suggestion
@input_error_decorator_factory(args_length = 6, message = "Invalid command. Usage: add_address <name> <street> <city> <postal_code> <country>")
```
Maybe we can delete args_length param in the future, but now it should be 6 in your case, as book is also function argumant